pyproject.toml

```diff
@@ -19,6 +19,9 @@ dataclasses = { version = "^0.7", python = ">=3.6, <3.7" }
 grpclib = "^0.3.1"
 jinja2 = { version = "^2.11.2", optional = true }
 protobuf = { version = "^3.12.2", optional = true }
+toml = { version = "0.10.1", optional = true }
```

Comment:
curious why the pinned version.

Reply:
No particular reason it was just the most recent

Reply:
The version itself is fine, though I'm curious why it's pinned without a `^`?